src/libs/providers.js

```diff
@@ -1,5 +1,5 @@
 export const allProviders = [
-  { key: 'fence', name: 'NHLBI BioData Catalyst Framework Services' },
-  { key: 'dcf-fence', name: 'NCI CRDC Framework Services' },
-  { key: 'anvil', name: 'NHGRI AnVIL Data Commons Framework Services' }
+  { key: 'fence', name: 'NHLBI BioData Catalyst Framework Services', expireTime: 30 },
+  { key: 'dcf-fence', name: 'NCI CRDC Framework Services', expireTime: 15 },
+  { key: 'anvil', name: 'NHGRI AnVIL Data Commons Framework Services', expireTime: 30 }
```

Comment:
nit: `expireTime` sounds like the date/time of expiration (which was the original use of the variable). Maybe something like `expiresAfter` or `daysValid` are a bit more precise?